ecosystem/typescript/sdk_v2/src/types/account_address.ts

```diff
@@ -0,0 +1,317 @@
+// Copyright © Aptos Foundation
```

Comment:
following this comment on HexData class, lets move this class to either `utils` or `code` folder https://github.com/aptos-labs/aptos-core/pull/9595/files#r1290573135